Jenkinsfile

```diff
@@ -74,7 +74,7 @@ pipeline {
                   -v "$WORKSPACE/.git:/go/src/github.com/docker/docker/.git" \
                   --name docker-pr$BUILD_NUMBER \
                   -e DOCKER_GITCOMMIT=${GITCOMMIT} \
-                  -e DOCKER_GRAPHDRIVER=vfs \
+                  -e DOCKER_GRAPHDRIVER=overlay2 \
                   -e DOCKER_EXECDRIVER=native \
```

Comment:
not related to this PR, but probably we can remove DOCKER_EXECDRIVER

Reply:
Yes, we should remove it

Reply:
#39606